frb_example/pure_dart/rust/.flutter_rust_bridge.yml

```diff
@@ -8,3 +8,4 @@ c_output: [c_output_path/c_output.h]
 extra_c_output_path:
   - c_output_path_extra/
 # dump: []
+no_use_bridge_in_method: true
```

Comment:
Not very sure, shall we enable this flag in this example? I guess yes, because this makes the API simpler to use (no need to carry a `bridge` instance here and there).

Reply:
I'm on the fence here, personally I wouldn't enable the flag for pure_dart as it's not the default but up to you.

Reply:
Yes that's a problem. 

An alternative may be, we somehow enhance our infrastructure to test more scenarios with various feature flags. IIRC our testing infra currently is quite limited about this - everyone who PR to make a change that only happens with *some* feature flag has to do hacky things like what you have done above...

Reply:
Fixtures may be the solution, but it would require a slightly more complicated PR. Luckily I should have more free time next month so I might look into it.

Reply:
Fixtures look reasonable (feel free to sketch the plan before implementing so that we can discuss it a bit more), and looking forward to your PR next month!

As for this current PR - if we are going to add those thorough tests in the future, maybe we can keep this PR simple and follow your suggestion:

> personally I wouldn't enable the flag for pure_dart as it's not the default